src/cosmic_ray/operators/util.py

```diff
@@ -19,3 +19,138 @@ def dec(cls):
         return cls
 
     return dec
+
+
+def dump_node(node, stdout=None):
```

Comment:
Isn't this part of some other PR?

Reply:
Yes, but since that other PR isn't merged, I need to checkout those part of code here from the other PR. git can merge this without problems.